source/ways-of-working/environments/external-access.html.md.erb

```diff
@@ -0,0 +1,85 @@
+---
+title: External Access in Demo
+last_reviewed_on: 2022-03-30
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+The demo environment is used to demonstrate new features to external stakeholders. 
+
+For guidance on how to deploy an application to the demo environment, see [Common Pipeline](https://hmcts.github.io/ways-of-working/common-pipeline/common-pipeline.html#building-infrastructure-in-the-demo-environment).
+
+There are three different ways in which an application in demo can be made available to external users. This is dependent on the ingress class used in your application.
+
+[Traefik](https://traefik.io/traefik/) is used as the ingress controller in HMCTS applications.
+
+The following traefik instances are available to use:
+
+| IngressClass | Default | Application configuration |
+|-|-|-|
+| traefik | Yes | Application will be reachable externally without VPN. End user will be required to authenticate to Azure AD before being redirected to the application |
+| traefik-private | No | Application will be available over VPN only |
+| traefik-no-proxy | No | Application will be reachable externally without VPN or authentication |
+
+### Protecting applications with Azure AD authentication
+    
+To ensure end users must authenticate to Azure AD before they can view the application in their web browser, you must also set the `enableOAuth` flag to `true`
+
+```yaml
+nodejs:
+    ingressClass: traefik
+    disableTraefikTls: false
+    enableOAuth: true
+```
+
+*In SDS, if you do not set an ingressClass flag in code, it will default to traefik.*
+
+The authentication with Azure AD is handled by an Azure AD App Registration in the [HMCTS Demo Azure AD tenant](https://portal.azure.com/hmctsexecdemo.onmicrosoft.com). To allow end users to be authenticated to your application, you must add a redirect URI for your application to the Azure AD App Registration.
+
+*If you are not currently a member of the HMCTS Demo Azure AD tenant, check with your team to see if anyone else is. Everyone in this tenant has permission to invite others.*
```

Comment:
This should be it's own heading, it's a common question so we should be able to link to it.

It should should have a link or an Azure AD CLI command to do it, (not sure if CLI got support yet)

Reply:
Figured it out. Needed to add --allow-no-subscriptions

Reply:
this seems not done

Reply:
Can you include a deep link to the page where they can invite the user from.

and say if no one has access raise a help request to platops who will add you and then you can add any others.

You can remove the italics as it looks weird now